django/contrib/admin/templates/admin/index.html

```diff
@@ -11,10 +11,17 @@
 
 {% block nav-sidebar %}{% endblock %}
 
+{% block content_title %}
+  <div class="admin-index-title">
+    {% if title %}<h1>{{ title }}</h1>{% endif %}
+    <input type="search" id="app-filter"
```

Comment:
Could we isolate this into a separated template to be included in both nav_sidebar.html and index.html?